src/js/components/Chart.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+  <div class="chart-container">
+    <svg id="chart" width="100%" height="310px">
+      <g>
+        <rect
+          class="cogs"
+          :x="X0"
+          :y="Y0"
+          :width="WIDTH"
+          :height="h_cogs_0 * scale"
+        />
+        <rect
+          class="sga"
+          :x="X0"
+          :y="Y0 + h_cogs_0 * scale"
+          :width="WIDTH"
+          :height="h_sga_0 * scale"
+        />
+        <template v-if="HEIGHT > expenses_0">
+          <rect
+            class="income"
+            :x="X0"
+            :y="Y0 + expenses_0 * scale"
+            :width="WIDTH"
+            :height="(HEIGHT - expenses_0) * scale"
+          />
+          <g
+            class="corner bottom-left"
+            :transform="`translate(${X0},${Y0 + HEIGHT * scale - 10})`"
```

Comment:
I think we may want to bump this in by ~ 1px because of the stroke the piece that is covering it? Seems to look a bit cut off without that little bump.

Reply:
@stacyk When I tried that locally, I get a very faint hint of a colored background on the wrong side of this corner. Feel free to play with it and push something you think is better.

Reply:
@jgerigmeyer I'll play with this in a second, but I am curious why the bottom left or bottom right corners aren't in the markup when there is a "negative" or whatever you want to call it when there is no green rectangle?

Reply:
@stacyk I understood that to be an intentional design choice -- it matches the mockup. I think it sort of highlights when there's no profit, and the expenses are more than revenue. I'm not too concerned about it unless @SondraE did not intend to do that, or if Cedric requests something different?